src/fluree/db/ledger/indexing.clj

```diff
@@ -275,15 +275,15 @@
       children)))
 
 (defn descendant?
```

Comment:
Can we add a docstring here to make clear which is the potential ancestor and which the potential descendant? Just to make it easier to use in the future when we've forgotten precisely which way the arguments were arranged : )

Reply:
And I see you moved this to the `db` lib, I'll move this comment there.